rancher2/structure_catalog_v2.go

```diff
@@ -26,7 +29,8 @@ func flattenCatalogV2(d *schema.ResourceData, in *ClusterRepo) error {
 	}
 	d.Set("resource_version", in.ObjectMeta.ResourceVersion)
 
-	d.Set("ca_bundle", string(in.Spec.CABundle))
+	encodedCABundle := base64.StdEncoding.EncodeToString(in.Spec.CABundle)
```

Comment:
Why are we encoding this? 

This is the clusterRepo 

```
apiVersion: catalog.cattle.io/v1
kind: ClusterRepo
metadata:
  creationTimestamp: '2024-01-29T17:06:01Z'
  generation: 3
  managedFields:
    - apiVersion: catalog.cattle.io/v1
      fieldsType: FieldsV1
      fieldsV1:
        f:spec:
          .: {}
          f:caBundle: {}
          f:clientSecret: {}
          f:url: {}
      manager: rancher
      operation: Update
      time: '2024-01-29T17:06:42Z'
    - apiVersion: catalog.cattle.io/v1
      fieldsType: FieldsV1
      fieldsV1:
        f:status:
          .: {}
          f:conditions: {}
          f:downloadTime: {}
          f:observedGeneration: {}
      manager: rancher
      operation: Update
      subresource: status
      time: '2024-01-29T17:06:42Z'
  name: testing
  resourceVersion: '186876'
  uid: 3dee071c-ec22-413a-a506-d9c1d10035b2
spec:
  caBundle: >-
    MIIB6TCCAY+gAwIBAgIIdt3GyRspkn4wCgYIKoZIzj0EAwIwRjEcMBoGA1UEChMTZHluYW1pY2xpc3RlbmVyLW9yZzEmMCQGA1UEAwwdZHluYW1pY2xpc3RlbmVyLWNhQDE3MDY1MjU5ODAwHhcNMjQwMTI5MTA1OTQwWhcNMjUwMTI4MTEwMjMzWjAkMRAwDgYDVQQKEwdkeW5hbWljMRAwDgYDVQQDEwdkeW5hbWljMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEkaT5FLdTzgs6yNaa8PltS785HCW2oNGFsm/4ZBgoMgQC63QpGA5+asRePP9tvSMmNf97zGc4T13cwkfu4RDYtKOBiDCBhTAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwHwYDVR0jBBgwFoAUZ2qHTZJTBsvTvg7FJDVSss8YB38wPQYDVR0RBDYwNIIJbG9jYWxob3N0ghVyYW5jaGVyLmNhdHRsZS1zeXN0ZW2HBH8AAAGHBKwRAAKHBKzUUUQwCgYIKoZIzj0EAwIDSAAwRQIhAKnE7zEoSuwX6XMUcffPC/LYpIqEGieQLTjMF8QQggYgAiB5eHGz7JY8biWQA4m5imWIRbTqZ9RPnflrKufZnCCBxg==
  clientSecret: null
  url: https://172.212.81.68/dashboard/
 ````
 
 The in.spec.Cabundle will be `MIIB6TCCAY+gAwIBAgIIdt3GyRspk...` Why can't we show this ? Why should we encode this?

Reply:
Let's lets:
```
output "some_NAME" {
  value = <the_catalog_resource>.ca_bundle
}
```

to validate this